arch/arm/src/sama5/hardware/_sama5d3x_memorymap.h

```diff
@@ -103,6 +103,8 @@
 #  define SAM_ISI_OFFSET         0x00034000 /* 0x00034000-0x00037fff: ISI */
 #  define SAM_SFR_OFFSET         0x00038000 /* 0x00038000-0x0003bfff: SFR */
                                             /* 0x0003c000-0x07ffffff: Reserved */
+
```

Comment:
No that the spaces adder after the right commcomments are incorrect.  They should be removed.

This is documented bug in nxstyle.  It is reporting that a blank line is needed, but that is a bug.  No blank line should be added:  See Issue #120 

Please remove these extra blank lines.  You are corrupted an otherwise correct file due to an error in nxstyle.

Reply:
@patacongo Will fix.

Re: nxstyle bug, I'm confused about what nxstyle errors to fix or not fix. It seems like if we are going to use a tool, it should report correctly. Is there someone in the nuttx community who knows the best way to fix this bug?

Reply:
@patacongo Fixed.